src/services/ui/src/features/package-actions/lib/index.ts

```diff
@@ -53,8 +59,19 @@ export const submitActionForm = async ({
     });
     alert.setBannerStyle("success");
     alert.setBannerShow(true);
-    alert.setBannerDisplayOn(path);
-    navigate({ path });
+    alert.setBannerDisplayOn(
+      locationState.from?.includes("dashboard") ? "/dashboard" : "/details",
+    );
+    const poller = new DataPoller({
+      interval: 1000,
+      pollAttempts: 20,
```

Comment:
do you have any recorded times for this interaction to succeed and redirect?? idk if 20 seconds is necessary

Reply:
I don't have any recorded. There are times where it took upwards of 15 seconds.

Reply:
Yeah, usually is but seems like there are weird times where it takes a little longer. Usually on like a brand new waiver or spa submission :(